public/apps/configuration/panels/user-list.tsx

```diff
@@ -45,6 +45,9 @@ import { buildHashUrl } from '../utils/url-builder';
 
 function dictView() {
   return (items: Dictionary<string>) => {
+    if (isEmpty(items)) {
+      return '-';
```

Comment:
nonblocking comment: this is the default placeholder for all empty strings. Probably we can consider making it a global constant. Leave it up to you.

Reply:
Good idea. Will do.

Reply:
Will do it in another PR.